govuk-one-login/authentication-frontend
# AUT-3825: IPV reverification redirects to change security codes

## What

Once a reverification with IPV is successful, we redirect to `PATH_NAMES.GET_SECURITY_CODES`. This allows MFA Reset with ID users to start resetting their MFA.

## How to review

1. Code Review
1. Deploy to dev
1. Sign in with email and password, then ask to reset MFA
1. See IPV stub and continue
1. See page to choose which MFA to reset

## Checklist









## Related PRs
